README.md

```diff
@@ -20,22 +20,68 @@ small set of sample data trained with intents like these:
 * `weather`
 * `capabilities`
 
-To see all the intents, open [`/training/car_intents.csv`](https://raw.githubusercontent.com/watson-developer-cloud/conversation-simple/master/training/car_intents.csv).
-
 These intents help the system to understand variations of questions and commands that you might submit.
 For example, if you say *"Wipers on"* or *"I want to turn on the windshield wipers"*, the system
 understands that in both cases your intent is the same and responds accordingly.
 
-## Getting Started
-If you used the **Deploy to Bluemix** button, skip this section and go to **Creating a workspace**.  
+<a name="bluemix">
+# Getting Started using Bluemix
+</a>
+
+## Before you begin
+1 Ensure that you have a [Bluemix account](https://console.ng.bluemix.net/registration/).
+
+2 Ensure that you have the necessary space available in your Bluemix account. This action deploys 1 application and 3 services.
+   * You can view this on your Bluemix Dashboard. Tiles will show what space you have available.
+   * For example, for Services & APIS
+
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;![](readme_images/services.PNG)
```

Comment:
HTML doesn't quite work within markdown -- that's the point of markdown. SOME html tags do, but many don't (and all of the indent/align ones do not).

CSS is a whole different ballgame, so would need discussed well after GA. None of us have done it to a markdown file.

Personally I think that example you sent looks horrible. I would never want to follow steps like that. I know why they used a table - and its Markdown. That isn't HTML or CSS. It's a table made with markdown -- because its the only way they found to indent/align anything. Having the written steps and the images all indented like equally is not easy to follow.

Markdown is not meant as a tool for layout, it is meant to simplify the process of writing for the web. So if we find ourselves stretching its feature set to do what we need, we might need to look at whether or not we're using the right tool.

I will schedule a meeting for after GA. I will include Justin, you, and Steve -- at a minimum. Let me know who else. Because we cannot continue to use readmes/markdown if we want it to be  very pretty.